shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/instance/metadata/proxy/ProxyInstanceMetaData.java

```diff
@@ -41,15 +43,23 @@ public ProxyInstanceMetaData(final String id, final int port) {
         this.id = id;
         ip = IpUtils.getIp();
         this.port = port;
+        checkProperty();
     }
     
     public ProxyInstanceMetaData(final String id, final String attributes) {
         this.id = id;
         String[] attributesList = attributes.split(DELIMITER);
         ip = attributesList[0];
         port = Integer.parseInt(attributesList[1]);
+        checkProperty();
     }
-    
+
+    private void checkProperty() {
```

Comment:
Sorry. The issue has already finished in https://github.com/apache/shardingsphere/pull/18853 .And the Proxy will failed to start up if port is invalid.